ruby-service/app/services/cache_coordinator.rb

```
@@ -0,0 +1,29 @@
+      request = Net::HTTP::Get.new(uri.path)
+      http.request(request)
+    end
+  end
+
+  def invalidate_node(node_address, cache_key)
+    stdout, _stderr, _status = Open3.capture3("redis-cli -h #{node_address} DEL #{cache_key}")
+    stdout.strip
+  end
+
+  def load_cache_config(config_path)
+    YAML.load(File.read(config_path))
```

RuboCop (1.87.0)

[convention] Use `load_file(config_path)` instead.

(Style/YAMLFileRead)